underfs/s3/src/main/java/alluxio/underfs/s3/S3UnderFileSystem.java

```diff
@@ -393,13 +393,17 @@ public boolean rename(String src, String dst) throws IOException {
   @Override
   public void setConf(Object conf) {}
 
-  // No ACL integration currently, no-op
   @Override
-  public void setOwner(String path, String user, String group) {}
+  public void setOwner(String path, String user, String group) throws IOException {
+    // Do not allow setting S3 owner via Alluxio yet.
+    throw new IOException("setOwner is not supported to S3 via Alluxio.");
```

Comment:
Good point, I've moved the error login within `FileSystemMaster`, rather than UnderFileSystem impl